src/Directory.Build.targets

```diff
@@ -44,7 +44,6 @@
     <!-- These are the valid BlobGroupBuildQuality values. -->
```

Comment:
Can't leave a comment on the exact line (41/42) - do we still need `_BlobGroupPreReleaseLabel` and `_BlobGroupPreReleaseIteration`, or are they used anywhere else?

Reply:
Good catch. I removed more unused properties.